src/meds_etl/omop.py

```diff
@@ -12,6 +12,7 @@
 import meds
 import polars as pl
 import pyarrow.parquet as pq
+from tqdm import tqdm
```

Comment:
Can you add this to the dependencies in pyproject.toml?

Reply:
Will do!